pkg/api/rest/rest.go

```diff
@@ -62,7 +62,7 @@ type Lister interface {
 	// This object must be a pointer type for use with Codec.DecodeInto([]byte, runtime.Object)
 	NewList() runtime.Object
 	// List selects resources in the storage which match to the selector.
-	List(ctx api.Context, label labels.Selector, field fields.Selector) (runtime.Object, error)
+	List(ctx api.Context, label labels.Selector, field fields.Selector, options *api.ListOptions) (runtime.Object, error)
```

Comment:
That's fine.  Delete and List are more similar than Gets to different subresources will be (proxy, logs, status are all different Get schemas)